src/PhpSpec/Process/ReRunner/PassthruReRunner.php

```diff
@@ -43,7 +43,8 @@ public function isSupported()
     {
         return (php_sapi_name() == 'cli')
             && $this->getExecutablePath()
-            && function_exists('passthru');
+            && function_exists('passthru')
+            && strtolower(substr(PHP_OS, 0, 3)) != "win";
```

Comment:
I guess I was thinking of this https://github.com/phpspec/phpspec/blob/master/src/PhpSpec/Console/ContainerAssembler.php#L303-L307